sdks/java/core/src/main/java/org/apache/beam/sdk/values/Row.java

```diff
@@ -197,7 +199,15 @@ public Byte getByte(int idx) {
   }
 
   /**
-   * Get a {@link TypeName#INT16} value by field index, {@link ClassCastException} is thrown if
+   * Get a {@link TypeName#BYTES} value by field index, {@link ClassCastException} is thrown if
+   * schema doesn't match.
+   */
+  public byte[] getBytes(int idx) {
+    return getValue(idx);
+  }
+
+  /**
+   * Get a {@link TypeName#INT16 16} value by field index, {@link ClassCastException} is thrown if
```

Comment:
Looks like there is an extra `16`? (This line should be unchanged.)

Reply:
done